dist/pom.xml

```diff
@@ -159,6 +159,48 @@
       <artifactId>spring-context</artifactId>
     </dependency>
 
+    <!-- Following are used for generating models from open api spec   -->
+    <dependency>
+      <groupId>javax.validation</groupId>
+      <artifactId>validation-api</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.openapitools</groupId>
+      <artifactId>jackson-databind-nullable</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
```

Comment:
:wrench: All these annotations module - maybe at the end we can check if they can be optional/provided? With annotations there's a chance this is the case, i.e. they're only used at compile time, and then we can reduce dependency bloat to some extent.

Reply:
:+1: Thanks. I will check it if we can scope it to compile time.